src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php

```diff
@@ -90,6 +90,14 @@ public function saveAction()
         $this->_redirect('*/*/index');
     }
 
+    public function loadAction()
+    {
+        $cron = new Mageone_Qps_Model_Cron();
+        $cron->getRules();
```

Comment:
Can you catch the exception here and if we are in admin context (not sure how to test) `Mage::getSession('admin/session')->addWarning(...)`

not 100% sure about the code, please check :D

Reply:
There is no exception to catch. It's already caught in the getRules-function